Microsoft.Toolkit.Uwp.UI.Animations/Xaml/AnimationSet.cs

```diff
@@ -34,14 +34,7 @@ public sealed class AnimationSet : DependencyObjectCollection
         /// <summary>
         /// Raised whenever the current animation ends.
```

Comment:
Maybe we should update the documentation here to "Raised whenever the current animation completes." to match the API wording.

Reply:
Right, makes perfect sense. Done in 9d4cdcea657bddca6c9ead4c74b3a0440e791a5f, thanks! 😄